helm/solr-operator/README.md

```diff
@@ -182,6 +182,7 @@ The command removes all the Kubernetes components associated with the chart and
 | serviceAccount.name | string | `""` | If `serviceAccount.create` is set to `false`, the name of an existing serviceAccount in the target namespace **must** be provided to run the Solr Operator with. This serviceAccount with be given the operator's RBAC rules. |
 | resources.limits | map[string]string |  | Provide Resource limits for the Solr Operator container |
 | resources.requests | map[string]string |  | Provide Resource requests for the Solr Operator container |
+| securityContext | map[string]string | `allowPrivilegeEscalation: false, runAsNonRoot: true` | Provide security context for the Solr Operator container |
```

Comment:
```suggestion
| securityContext | object | `allowPrivilegeEscalation: false, runAsNonRoot: true` | Provide security context for the Solr Operator container |
```

the securityContext is an object, that can have values that aren't strings. So we shouldn't use `map[string]string` as the type.